### DISCUSSION TITLE
Fomantic UI + Bragndrop

### DISCUSSION THREAD

**Discussion by ayoob-lfc** [fomantic/Fomantic-UI#1917] [Showroom]
Hello,

We previously integrated Semantic UI in our [Bragndrop Editor](https://editor.bragndrop.com/?url=editor&fw=7) so that you can quickly create apps or play around with the components using the drag'n'drop editor. 

We've started the process of integrating Fomantic-UI (as in testing it). If you are interesting in playing around with it, you can do it via:

1. Download and save the [Fomantic-UI](https://raw.githubusercontent.com/hamari-agency/bragndrop-developer-help/master/src/framework/fomanticui2.js) Setting file to your computer
2. Go to [hacked.bragndrop.com](https://hacked.bragndrop.com)
3. Click on **Import**
4. Click on **Open your file** and select the downloaded **fomanticui2.js** file
5. Once the file has been loaded, hit the **Update** button (top right of the left panel)
6. Enjoy

![image](https://user-images.githubusercontent.com/46787382/111508191-56938880-8743-11eb-9b05-81634ff33220.png)

**Comment by hugopeek** [comment]
Thanks @ayoob-lfc, this is really cool!!
